user_guide_src/source/incoming/controllers.rst

```diff
@@ -412,6 +412,8 @@ without route definitions. The auto-routing is disabled by default.
 
 .. important:: Auto Routing (Legacy) routes a HTTP request with **any** HTTP method to a controller method.
 
+.. important:: If Auto Routing (Legacy) doesn't find the controller, it will throw page not found exception before the filter executes in version **4.5**.
```

Comment:
```suggestion
.. important:: Since v4.5.0, if Auto Routing (Legacy) doesn't find the controller,
    it will throw ``PageNotFoundException`` exception before the Controller Filters
    execute.
```

Reply:
Done.